compiler/luci-interpreter/src/kernels/OneHot.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Added TF copyright because implementation  is refactored copy of:

https://github.com/tensorflow/tensorflow/blob/bd22aa9a24b1cb49a4e0848e2bf35fd1c66f49a6/tensorflow/lite/kernels/one_hot.cc#L63-L100